src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
 // write your code here
+const trs = document.querySelectorAll('table tr');
+
+for (const tr of trs) {
+  const p = tr.cells[1].cloneNode(true);
+
+  tr.append(p, tr.cells[tr.cells.length - 1]);
```

Comment:
The `append` method is being used incorrectly here. The method should append the cloned cell `p` to the row `tr`, but the second argument `tr.cells[tr.cells.length - 1]` is unnecessary and could lead to unexpected behavior. Consider appending only `p` to the row.